client/src/components/CheckBox/CheckBox.tsx

```diff
@@ -10,10 +11,10 @@ import "./CheckBox.scss";
 
 export interface CheckBoxProps {
   disabled?: boolean; // greys out checkbox and make it non-clickable if true
-  onClick: Function; // (id) => {}
-  id: string | number;
+  onClick?: Function; // (id) => {}
+  id?: string | number;
   label: string; // required text for checkbox
-  active: boolean; // required, equivalent of 'checked'
+  active?: boolean; // required, equivalent of 'checked'
   color?: string; // defaults to primaryColor
   isSolidBox: boolean; // defaults to false
   container_style?: Object; // style for div, containing checkbox and label
```

Comment:
Looks like this should be `React.CSSProperties`

Reply:
Didn't know that existed!